README.rst

```diff
@@ -18,15 +18,36 @@ Installation
 Usage
 =====
 
-::
+Running a match:
+
+.. code-block:: python
+
+   >>> import axelrod_fortran as axlf
+   >>> import axelrod as axl
+   >>> p1 = axlf.Player('k31r')
+   >>> p2 = axlf.Player('k33r')
+   >>> match = axl.Match((p1, p2), turns=5)
+   >>> match.play()
+   [(C, C), (C, C), (C, D), (C, D), (C, C)]
+
+Reproducing Axelrod's second tournament:
+
+.. code-block:: python
+
+   import axelrod as axl
+   from axelrod_fortran.strategies import characteristics
+   from axelrod_fortran.player import Player
+
+   players = [Player(name) for name in characteristics.keys()
+              if characteristics[name]["original_rank"] is not None]
+
+   print(len(players), "players")
 
-    >>> import axelrod_fortran as axlf
-    >>> import axelrod as axl
-    >>> p1 = axlf.Player('k31r')
-    >>> p2 = axlf.Player('k33r')
-    >>> match = axl.Match((p1, p2), turns=5)
-    >>> match.play()
-    [(C, C), (C, C), (C, D), (C, D), (C, C)]
+   tournament = axl.Tournament(players, repetitions=100)
```

Comment:
Actually, feel free to leave it for now, I'm going to open a PR to axelrod that will make it possible to pass what a player knows in a tournament (we can currently do that in a match). Then we can just fix this.